new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiConfigMap.java

```diff
@@ -332,13 +331,13 @@ public void testCreateMiiConfigMapDomain() {
 
     logger.info("Check admin service {0} is created in namespace {1}",
         adminServerPodName, domainNamespace);
-    checkServiceCreated(adminServerPodName, domainNamespace);
+    checkServiceExists(adminServerPodName, domainNamespace);
```

Comment:
In all It classes, the service existence check should be done right after admin pod ready check. Otherwise  if there are errors in creating the service the assertion will fail at waiting for managed server to come up when the actual problem was in the service creation of admin server pod.

Reply:
done